autorest/adal/token.go

```diff
@@ -54,6 +55,9 @@ const (
 
 	// metadataHeader is the header required by MSI extension
 	metadataHeader = "Metadata"
+
+	// imdsEndpoint is the well known endpoint for getting MSI authentications tokens
+	imdsEndpoint = "http://169.254.169.254/metadata/identity/oauth2/token"
```

Comment:
>imdsEndpoint [](start = 1, length = 12)

also not sure if this is confusing or not, maybe still name it MSI endpoint as this is what this endpoint is ( just that is part of IMDS now )